Sources/CorePayments/Eligibility/SupportedPaymentMethods.swift

```diff
@@ -0,0 +1,7 @@
+import Foundation
+
+enum SupportedPaymentMethodsType: String {
```

Comment:
Do we want to lowercase these as well or keep as is since it's internal?

Reply:
Great idea!!! Done: [b422115](https://github.com/paypal/paypal-ios/pull/286/commits/b422115c83244210d4026c0d7ff500d004fa41de)